Data_Collection/CRON_scripts/make_forecast.py.orig

```diff
@@ -0,0 +1,529 @@
+#!/usr/bin/env python3
```

Comment:
do we need to track this file? looks like a backup file

Reply:
Well, this line is created automatically in spyder (the IDE that I'm using).

Reply:
I mean, the `make_forecast.py.orig` looks like a backup file, if it is correct, do we need to track that down by git? normally, we don't add to git a copy of an old version, because git track all changed files, and if we want, we can just go to the history and get what we need.

Reply:
Sorry, I thought you were talking about the file `colombia_data.py`. I will see if it's everything right with this file. Maybe some problem happened because of differences in the files in the two branchs.

Reply:
the .orig is generated by git on some merge conflicts. I would just delete.